syft/linux/release.go

```diff
@@ -30,6 +34,9 @@ func (r *Release) String() string {
 	if r.Version != "" {
 		return r.ID + " " + r.Version
 	}
+	if r.VersionID != "" {
+		return r.ID + " " + r.VersionID
+	}
 
-	return r.ID + " " + r.VersionID
+	return r.ID + " " + r.BuildID
```

Comment:
I actually never questioned why this was the way it was written previously, but do you have a source for why we're making the change here?

Reply:
For this change we are simply just adding a new fallback on `BuildID` while the existing usage should default to `VersionID` as previously.